crates/config/src/config.rs

```diff
@@ -228,154 +229,154 @@ impl TestNodeConfig {
                     .to_string()
                     .green()
             );
-            tracing::info!(
+            sh_println!(
                 "Batch Overhead L1 Gas:     {}",
                 fd.fee_model_config_v2
                     .batch_overhead_l1_gas
                     .to_string()
                     .green()
             );
-            tracing::info!(
+            sh_println!(
                 "Max Gas Per Batch:         {}",
                 fd.fee_model_config_v2.max_gas_per_batch.to_string().green()
             );
-            tracing::info!(
+            sh_println!(
                 "Max Pubdata Per Batch:     {}",
                 fd.fee_model_config_v2
                     .max_pubdata_per_batch
                     .to_string()
                     .green()
             );
-            println!("\n");
+            sh_println!("\n");
         } else {
-            tracing::info!("Network Configuration");
-            tracing::info!("========================");
-            tracing::info!(
+            sh_println!("Network Configuration");
+            sh_println!("========================");
+            sh_println!(
                 "Chain ID: {}",
                 self.chain_id
                     .unwrap_or(TEST_NODE_NETWORK_ID)
                     .to_string()
                     .green()
             );
-            println!("\n");
+            sh_println!("\n");
         }
-        tracing::info!("Gas Configuration");
-        tracing::info!("========================");
-        tracing::info!(
+        sh_println!("Gas Configuration");
+        sh_println!("========================");
+        sh_println!(
             "L1 Gas Price (gwei):               {}",
             format_gwei(self.get_l1_gas_price().into()).green()
         );
-        tracing::info!(
+        sh_println!(
             "L2 Gas Price (gwei):               {}",
             format_gwei(self.get_l2_gas_price().into()).green()
         );
-        tracing::info!(
+        sh_println!(
             "L1 Pubdata Price (gwei):           {}",
             format_gwei(self.get_l1_pubdata_price().into()).green()
         );
-        tracing::info!(
+        sh_println!(
             "Estimated Gas Price Scale Factor:  {}",
             self.get_price_scale().to_string().green()
         );
-        tracing::info!(
+        sh_println!(
             "Estimated Gas Limit Scale Factor:  {}",
             self.get_gas_limit_scale().to_string().green()
         );
-        println!("\n");
+        sh_println!("\n");
 
-        tracing::info!("Genesis Timestamp");
-        tracing::info!("========================");
-        tracing::info!("{}", self.get_genesis_timestamp().to_string().green());
-        println!("\n");
+        sh_println!("Genesis Timestamp");
+        sh_println!("========================");
+        sh_println!("{}", self.get_genesis_timestamp().to_string().green());
+        sh_println!("\n");
```

Comment:
Something I have wanted to do for a while is to refactor this file to print using raw strings instead, like this:
```rust
sh_print!(
    r#"

Genesis Timestamp
========================
{}

    "#,
    self.get_genesis_timestamp().to_string().green()
);
```

The code indents won't be pretty but it will be easy to see what each table looks like (and ensure that fields are properly aligned etc). Can be done in a separate PR ofc